src/pages/AdminDashboard/UserList/AdminToolbar/AdminToolbar.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import { IconButton, Grid, Popper, Paper } from '@material-ui/core';
+import FilterListIcon from '@material-ui/icons/FilterList';
+
+import CheckBox from '../../../../components/CheckBox';
+import SearchToolbar from '../../../../components/SearchToolbar';
+
+import { mockData, statusTags } from '../../data';
+
+const useStyles = makeStyles((theme) => ({
+    root: {},
+    filterPopper: {
+        maxWidth: '300px',
+        marginRight: theme.spacing(1),
+    },
+}));
+
+type userInfo = {
+    id: number;
+    name: string;
+    status: string;
+    timeStamp: number;
+};
+
+export interface Props {
+    onLoadUsers: (setHandler: Array<userInfo>) => void;
+    filterLabel?: string;
+}
+
+const AdminToolbar = ({ onLoadUsers, filterLabel }: Props) => {
+    const classes = useStyles();
+    const [filterAnchorEl, setFilterAnchorEl] = useState(null);
+    const open = Boolean(filterAnchorEl);
+    const [enteredFilter, setEnteredFilter] = useState<string>('');
+    const [enteredFilterTags, setEnteredFilterTags] = useState<Array<string>>(
+        []
+    );
+
+    const inputRef = useRef();
+
+    const filterClickHandler = (event: React.MouseEvent) => {
+        open ? setFilterAnchorEl(null) : setFilterAnchorEl(event.currentTarget);
+    };
+
+    useEffect(() => {
+        //* API FETCH SHOULD GO HERE
```

Comment:
Just as an fyi, you can do your api fetch here like you would do it in production and define mocks so that way your request has data.  I can show you this on Monday.